src/app/forms/templates/general/hgv-trl-brakes.template.ts

```diff
@@ -1,8 +1,8 @@
 import { ValidatorNames } from '@forms/models/validators.enum';
 import { FormNode, FormNodeTypes } from '../../services/dynamic-form.types';
 
-export const TrlBrakes: FormNode = {
-  name: 'trlBrakesSection',
+export const HgvAndTrlBrakesTemplate: FormNode = {
```

Comment:
HGVs don't have any brake values, only PSV and TRL:  https://dvsa.atlassian.net/wiki/spaces/HVT/pages/18286839/View+Technical+Record

Reply:
I checked [this ticket](https://dvsa.atlassian.net/browse/CB2-5701) and got confused by its first line of description. You are right. The brakes section for HGVs is not removed.